src/components/common/YourCompany/YourCompany.vue

```diff
@@ -467,7 +465,7 @@ export default class YourCompany extends Mixins(
       return this.getResource.changeData.nameChangeOptions
         .filter(option => option !== CorrectionTypes.CORRECT_NAME_TO_NUMBER)
     }
-
+    if (this.isEntityTypeFirm) return this.getResource.changeData.nameChangeOptions
     return this.getResource.changeData.nameChangeOptions
```

Comment:
Lines 468 and 469 return the same value, so you don't need the first one.

Reply:
ok

Reply:
Did you have something else in mind? If so, please add a code comment explaining it.

Reply:
No, I didn't. This makes sense now.